frontend/public/components/utils/resource-log.jsx

```diff
@@ -45,6 +50,26 @@ const LogControls = ({dropdown, onDownload, toggleFullscreen, isFullscreen, stat
       {dropdown && <div className="co-toolbar__item">{dropdown}</div>}
     </div>
     <div className="co-toolbar__group co-toolbar__group--right">
+      {!_.isEmpty(podLogLinks) && _.map(_.sortBy(podLogLinks, 'metadata.name'), (link, i) => {
+        const namespace = resource.metadata.namespace;
+        const namespaceFilter = link.spec.namespaceFilter;
+        const hrefTemplate = link.spec.hrefTemplate;
+        const labels = resource.metadata.labels;
+        // if namespaceFilter, do not output link if namespace does not match namespaceFilter regex
+        if (namespaceFilter && namespace.search(namespaceFilter) === -1) {
+          return null;
+        }
+        const url = hrefTemplate
+          .replace('${podName}', encodeURIComponent(resource.metadata.name))
+          .replace('${podUID}', encodeURIComponent(resource.metadata.uid))
+          .replace('${podContainerName}', encodeURIComponent(containerName))
+          .replace('${podNamespace}', encodeURIComponent(namespace))
+          .replace('${podLabels}', _.map(_.keys(labels), k => `${encodeURIComponent(k)}=${encodeURIComponent(labels[k])}`).join('|'));
```

Comment:
Per our conversation, I'd suggest

* `resourceName`
* `resourceUID`
* `containerName`
* `resourceNamespace`
* `podLabels`

Reply:
I'd JSON stringify the labels

```suggestion
          .replace('${podLabels}', JSON.stringify(labels));
```